crypto/x509/x_pubkey.c

```diff
@@ -493,3 +522,13 @@ int X509_PUBKEY_eq(const X509_PUBKEY *a, const X509_PUBKEY *b)
         return -2;
     return EVP_PKEY_eq(pA, pB);
 }
+
+int X509_PUBKEY_get0_libctx(OPENSSL_CTX **plibctx, const char **ppropq,
+                            const X509_PUBKEY *key)
+{
+    if (plibctx)
```

Comment:
Is it worth a check `if (key == NULL) return 0` here ?

Reply:
Maybe in a fixup PR?  `X509_PUBKEY_get0_param` doesn't make this check either...

Reply:
explicit checks against NULL are what the style says.  probably several of those to fix here.